lib/wings/converter_value_mapper.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Wings
+  ##
+  # A base value mapper for converting property values in the
+  # `Valkyrie` type system to `ActiveFedora`/`ActiveTriples` type
+  #
+  # This top level matcher has registered several internal mappers which handle
+  # indivdual value types from the source data.
+  class ConverterValueMapper < ::Valkyrie::ValueMapper; end
+
+  class NestedResourceArrayValue < ::Valkyrie::ValueMapper
+    ConverterValueMapper.register(self)
+    def self.handles?(value)
+      value.last.is_a?(Array) && value.last.map { |x| x.try(:class) }.include?(Hash)
+    end
+
+    def result
+      ["#{value.first}_attributes".to_sym, values]
+    end
+
+    def values
+      value.last.map do |val|
+        calling_mapper.for([value.first, val]).result
+      end.flat_map(&:last)
+    end
+  end
+
+  class NestedResourceValue < ::Valkyrie::ValueMapper
+    ConverterValueMapper.register(self)
+    def self.handles?(value)
+      value.last.is_a?(Hash)
+    end
+
+    def result
+      # [value.first, ActiveFedoraConverter.new(resource: value.last).convert]
```

Comment:
If this is a comment, then it should likely be expanded please.  Otherwise, I would please request that it be removed.

Reply:
I just moved this here, so I'm not sure the original purpose of this comment.  I suspect it is left over kruft from the original commit.  @laritakr Do you agree that this is ok to delete?

Reply:
👍